immuni_exposure_ingestion/monitoring/celery.py

```diff
@@ -0,0 +1,38 @@
+from prometheus_client import Counter, Gauge
+
+from immuni_common.monitoring.core import NAMESPACE, Subsystem
+
+KEYS_PROCESSED = Counter(
+    namespace=NAMESPACE,
+    subsystem=Subsystem.API.value,
+    name="keys_processed",
+    documentation="Total number of keys processed.",
+)
+
+BATCHES_CREATED = Counter(
+    namespace=NAMESPACE,
+    subsystem=Subsystem.API.value,
+    name="batches_created",
+    documentation="Total number of Batches created.",
+)
+
+UPLOADS_ENQUEUED = Gauge(
+    namespace=NAMESPACE,
+    subsystem=Subsystem.API.value,
+    name="uploads_enqueued",
+    documentation="Total number of Uploads still to be processed after completing a batch",
```

Comment:
Should we add this should never happen? While reading the metric description would come handy to understand which visualization/alert to associate it to.

Reply:
Also, sentence to end with `.`.